code/controllers/CMSPageHistoryController.php

```diff
@@ -358,7 +358,7 @@ public function ShowVersionForm($versionID = null) {
 	/**
 	 * @param int $versionID
 	 * @param int $otherVersionID
-	 * @return Form
+	 * @return mixed Form|boolean|SS_HTTPResponse
```

Comment:
Ah! well - I don't think this should be returning a `SS_HTTPResponse`, should it??